flocker/acceptance/scripts/exitinghttp.py

```diff
@@ -4,14 +4,19 @@
 
 from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
 import os
+import time
 
 
 class Handler(BaseHTTPRequestHandler):
     def do_GET(s):
         s.send_response(200)
+        s.send_header("content-length", "2")
         s.end_headers()
         s.wfile.write(b"hi")
+        s.wfile.flush()
         s.wfile.close()
+        # Try to ensure enough time for bytes to make it out before exiting:
+        time.sleep(0.1)
```

Comment:
I'm sceptical that this sleep is needed. Do you have experimental evidence? If not, can we please try it with just `flush` and using `sys.exit()` instead of `os._exit(1)` and see what happens?